src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/SupplierContact.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[SupplierContact.sql]
+(
+    [ContactId] INT NOT NULL PRIMARY KEY, 

```

Comment:
`ContactId` probably ought to be an `IDENTITY(1, 1)` column (which will match the address table added by @RuthBerry96).